### ISSUE TITLE
用CTC直接微调效果非常差

### ISSUE THREAD

**Issue by zyh3826** [TencentGameMate/chinese_speech_pretrain#39]
在common voice上使用huggingface的`Wav2Vec2ForCTC`进行微调，模型加载
```python
model = Wav2Vec2ForCTC.from_pretrained(
                                        model_path,
                                        ctc_loss_reduction='mean',
                                        pad_token_id=tokenizer.pad_token_id,
                                        vocab_size=len(tokenizer))
model.freeze_feature_extractor()
```
loss从160直接干到4就不降了，预测的时候输入一句话，其输出向量每一行都是一样的，也就是说只输出同一个字，如图
train loss
![image](https://github.com/TencentGameMate/chinese_speech_pretrain/assets/31238754/31ad8c69-af65-4047-8536-fc1e078b1054)
eval loss
![image](https://github.com/TencentGameMate/chinese_speech_pretrain/assets/31238754/f09411ea-924a-4fc7-9ccd-e91c008843cf)
logits
![image](https://github.com/TencentGameMate/chinese_speech_pretrain/assets/31238754/da8a3d61-38ef-483c-92a8-81e663f56451)
pred_id
![image](https://github.com/TencentGameMate/chinese_speech_pretrain/assets/31238754/beea73ba-3552-44e3-a031-c6a73bba58c3)
怎么调都无法成功。
但是在common voice上用`facebook/wav2vec2-large-xlsr-53`微调是有效果的
![image](https://github.com/TencentGameMate/chinese_speech_pretrain/assets/31238754/062fc86b-5bf4-425f-bef5-c1393c121368)

**Comment by LiuShixing** [comment]
hugging face使用我用的是Wav2Vec2Model，没有用Wav2Vec2ForCTC，不知道是不是因为这个。另外，common voice是多语言吧，我们的模型只在中文上训练的。其他语言能力未知

**Comment by zyh3826** [comment]
> hugging face使用我用的是Wav2Vec2Model，没有用Wav2Vec2ForCTC，不知道是不是因为这个。另外，common voice是多语言吧，我们的模型只在中文上训练的。其他语言能力未知

common voice的中文数据，2GB的那个，Facebook那个也是没有CTC，自己做词表，然后接CTC微调

**Comment by lizongyao123** [comment]
@zyh3826 你直接CTC微调最后成功了吗

**Comment by zyh3826** [comment]
> @zyh3826 你直接CTC微调最后成功了吗

没有，换fb的模型了

**Comment by sugermano** [comment]
> > hugging face使用我用的是Wav2Vec2Model，没有用Wav2Vec2ForCTC，不知道是不是因为这个。另外，common voice是多语言吧，我们的模型只在中文上训练的。其他语言能力未知
> 
> common voice的中文数据，2GB的那个，Facebook那个也是没有CTC，自己做词表，然后接CTC微调

你好，我想请教一下怎么自己做词汇表？？

**Comment by zyh3826** [comment]
> > > hugging face使用我用的是Wav2Vec2Model，没有用Wav2Vec2ForCTC，不知道是不是因为这个。另外，common voice是多语言吧，我们的模型只在中文上训练的。其他语言能力未知
> > 
> > 
> > common voice的中文数据，2GB的那个，Facebook那个也是没有CTC，自己做词表，然后接CTC微调
> 
> 你好，我想请教一下怎么自己做词汇表？？

时间有点久了，忘记怎么操作的了

**Comment by kirayomato** [comment]
遇到了一样的问题，loss下降到一定值以后就不再下降，输出全是空白
